lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -197,6 +197,22 @@ def instructor_dashboard_2(request, course_id):
     if certs_enabled and access['admin']:
         sections.append(_section_certificates(course))
 
+    def get_course_blocks(course_key, category):
+        """
+        Retrieve all XBlocks in the course for a particular category.
+
+        Returns only XBlocks that are published and haven't been deleted.
```

Comment:
Please, use a first person here, or use a third person on the line 202. @andrey-canon